app/controllers/users/omniauth_callbacks_controller.rb

```
@@ -49,12 +49,12 @@ def find_for_oauth_by_email(auth_attr)
       user = User.new(:email => auth_attr[:email], 
                       :password => Devise.friendly_token[0,20])
       user.save
```

here you should use `save!` to avoid a silent error or use `User.create` in the line 49